src/Household.tsx

```diff
@@ -63,15 +75,33 @@ export default function Household(props: Props) {
                       ></i>
                       {result.person.name + " "}
                     </span>{" "}
-                    {" should quarantine from "}{" "}
+                    {` should ${
+                      result.infected ? "isolate" : "quarantine"
+                    } from `}{" "}
                     {format(result.startDate, "MM/dd/yyyy")}
                     {" until "} {format(result.endDate, "MM/dd/yyyy")}
                     {"."}
+                    {result.infected && (
+                      <div className="pa2">
+                        If showing symptoms, continue to isolate until 24 hours
```

Comment:
I know this is totally lame, but showing it only if the person has reported symptoms needs us to replace the `infected` boolean with  a `CovidEventName` type field.